cdci_data_analysis/analysis/email_helper.py

```diff
@@ -268,6 +269,10 @@ def send_job_email(
                                              'attachment',
                                              filename="api_code.py")
 
+    status_details_generated = False
+    if status_details is not None:
```

Comment:
so if it contains exception, it will be sent in the email, and not the nice message?
It should be the nice message and exception details.

Reply:
are you referring to a log message? if so that is generated within the dedicated function within the `Instrument` class

Reply:
yes. Yes, it is generated there. But what will be sent if the generated message contains exception?

Reply:
in the email it is mentioned that it contains an empty product